pkg/terraform/install.go

```diff
@@ -28,8 +30,25 @@ func (m *Mixin) Install() error {
 		step.Flags = append(step.Flags, builder.NewFlag("input=false"))
 	}
 
-	for _, k := range sortKeys(step.Vars) {
-		step.Flags = append(step.Flags, builder.NewFlag("var", fmt.Sprintf("'%s=%s'", k, step.Vars[k])))
+	if step.CreateVarFile {
+		vf, err := m.FileSystem.Fs.Create(path.Join(m.WorkingDir, m.TerraformVarsFilename))
+		if err != nil {
+			return err
+		}
+		defer vf.Close()
+		vbs, err := json.Marshal(step.Vars)
+		if err != nil {
+			return err
+		}
+		_, err = vf.Write(vbs)
+		if err != nil {
+			return err
+		}
+		fmt.Printf("TF var file created:\n%s\n", string(vbs))
```

Comment:
I wonder if we should remove this log line? Perhaps to avoid printing sensitive parameter values?  Or perhaps a compromise would be to conditionally print if [debug](https://github.com/getporter/terraform-mixin/blob/main/cmd/terraform/main.go#L34) is set to `true`?